processing/src/main/java/org/apache/druid/segment/ColumnSelectorBitmapIndexSelector.java

```diff
@@ -157,14 +158,16 @@ public void close() throws IOException
   }
 
   @Override
-  public boolean hasMultipleValues(final String dimension)
+  public ColumnCapabilities.Capable hasMultipleValues(final String dimension)
   {
     if (isVirtualColumn(dimension)) {
       return virtualColumns.getVirtualColumn(dimension).capabilities(dimension).hasMultipleValues();
     }
 
     final ColumnHolder columnHolder = index.getColumnHolder(dimension);
-    return columnHolder != null && columnHolder.getCapabilities().hasMultipleValues();
+    return columnHolder != null
+           ? columnHolder.getCapabilities().hasMultipleValues()
+           : ColumnCapabilities.Capable.UNKNOWN;
```

Comment:
Ah, I changed `ExpressionFilter` to check `isMaybeTrue` to handle unknown, so i think unknown is fine here.

Reply:
I think the code you have is correct, but it's not going to do the optimized path (return `supportsBitmapIndex = true`) for a missing column, even though it would be okay to do it.

Reply:
Ah, I see what you are saying, but in your first comment you meant `Capable.FALSE`, so that [this code path is taken](https://github.com/apache/druid/blob/master/processing/src/main/java/org/apache/druid/segment/filter/ExpressionFilter.java#L118) to use the empty bitmap. Updated the PR and left a comment in the code to explain what is going on there

Reply:
Sorry, you're right, I meant `Capable.FALSE`. Thanks.